bindings/python/src/decoders.rs

```diff
@@ -21,7 +21,7 @@ use super::error::ToPyResult;
 ///
 /// This class is not supposed to be instantiated directly. Instead, any implementation of
 /// a Decoder will return an instance of this class when instantiated.
-#[pyclass(dict, module = "tokenizers.decoders", name=Decoder)]
+#[pyclass(dict, module = "tokenizers.decoders", name = "Decoder", subclass)]
```

Comment:
1. The `name` change is a breaking change from pyo3.
2. Added `subclass` because since pyo3 0.13 classes are not inheritable by default unless `subclass` attribute is set: https://pyo3.rs/v0.16.2/migration.html#runtime-changes-to-support-the-cpython-limited-api